cometr/global_metrics/FSC_toupy.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import os
+import time
+
+import numpy as np
+from beartype import beartype
+
+import toupy
+
+from cometr.global_metrics.Metric import Metric
+
+
+class FSC(Metric):
```

Comment:
Same as Neto's PR. 

for readability of the code and self documentation you should consider using more descriptive class or function names. This will help when the class is called to create the object to be clear what its doing. for example:

**with the above class name**

```
from FSC_toupy import FSC
object = FSC(metric)
```

however with a clear and descriptive class name its easier to scan through the code

```
from FSC_toupy import FourierShellCorrelation
object = FourierShellCorrelation(metric)
```

then its up to the user if they want to use it like this and self document their code so its easy to read or if they want to shorten it by saying something like `from FSC_toupy import FourierShellCorrelation as FSC`

In the above example its hard to see but imagine you import the FSC function and don't create the object for 300 lines of code. A reviewer or someone wanting to use your code would have to scroll way up to find the import and then go to a completely different file just to find out what FSC is short for. By being descriptive with your class and function names, you allow the code to explain itself to the user, even if the explanation is all in the imports and it gets abbreviated at that point if that makes sense?
Sort of like when you're writing a document. You wouldn't use an abbreviation in that document without explaining what it means the first time you use it.

As you know though, this is all preference and completely up to the coder (you). This is just how I think a package is best structured to make people enjoy and want to use it. Drop me a message if you have any questions.